extensions/wikidata/module/MOD-INF/controller.js

```diff
@@ -0,0 +1,76 @@
+
+importPackage(org.openrefine.wikidata.commands);
+
+/*
+ * Function invoked to initialize the extension.
+ */
+function init() {
+    var RefineServlet = Packages.com.google.refine.RefineServlet;
+    RefineServlet.registerClassMapping(
+            "org.openrefine.wikidata.operations.SaveWikibaseSchemaOperation$WikibaseSchemaChange",
+            "org.openrefine.wikidata.operations.SaveWikibaseSchemaOperation$WikibaseSchemaChange");
+    RefineServlet.registerClassMapping(
+            "org.openrefine.wikidata.operations.PerformWikibaseEditsOperation$PerformWikibaseEditsChange",
+            "org.openrefine.wikidata.operations.PerformWikibaseEditsOperation$PerformWikibaseEditsChange");
+    
+    RefineServlet.cacheClass(Packages.org.openrefine.wikidata.operations.SaveWikibaseSchemaOperation$WikibaseSchemaChange);
+    RefineServlet.cacheClass(Packages.org.openrefine.wikidata.operations.PerformWikibaseEditsOperation$PerformWikibaseEditsChange);
+
+    /*
+     *  Attach a Wikibase schema to each project.
+     */
+    Packages.com.google.refine.model.Project.registerOverlayModel(
```

Comment:
To EACH project? should overlay model always optional?

Reply:
It just means we add an empty one by default, which does not cost anything… I believe that's how it's done in the Freebase and RDF extensions. I don't see how you would "select" which projects to enable?